app/Services/Mail/ContactService.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Services\Mail;
+
+use Illuminate\Mail\Mailer;
+use Illuminate\Mail\Message;
+use Psr\Log\LoggerInterface;
+
+class ContactService
```

Comment:
From my practice and clean code it is recommended to avoid suffixes like `Service`, `Manager` and `Handler`. That's why I used the Mailer name to have specific name. Think about it for future.

Reply:
I see. Thanks ! Changed it's name from Mail/ContactService.php to ContactMailer.php. As well removed the Mail/ folder. Seems a little redundant for now. I'll add it later when I have more services.